packages/filebrowser-extension/src/index.ts

```diff
@@ -347,6 +350,23 @@ function addCommands(app: JupyterLab, tracker: InstanceTracker<FileBrowser>, bro
     mnemonic: 0
   });
 
+  commands.addCommand(CommandIDs.openTab, {
+    execute: () => {
+      const widget = tracker.currentWidget;
+
+      if (!widget) {
+        return;
+      }
+
+      return Promise.all(toArray(map(widget.selectedItems(), item => {
+        return commands.execute('docmanager:open-tab', { path: item.path });
+      })));
+    },
+    iconClass: 'jp-MaterialIcon jp-AddIcon',
```

Comment:
`jp-AddIcon` is currently used as a placeholder, so we might need a new icon (or remove it) if this one is not appropriate.

Reply:
Honestly, the `jp-AddIcon` looks okay to me. A plus symbol for new browser tab seems pretty common when I looked around. The only issue is that we have talked about adding a few other items to the filebrowser context menu like "New File" and "New Directory", and those may also want a similar icon. Did you have any other ideas about what might work?

Reply:
True, Firefox also uses a plus symbol to open a new tab (although it's blank). The alternatives would be:
- no icon
- create a new icon similar to the [external-link-alt](https://fontawesome.com/icons/external-link-alt?style=solid) from Font Awesome

For the New Directory, there is already the "New Folder" icon that could be reused in the filebrowser context menu:
![image](https://user-images.githubusercontent.com/591645/38509885-7e98c912-3c23-11e8-9dbc-33bb34147aac.png)
But there doesn't seem to be any existing icon for "New File". I think something similar to the "Copy" icon but with a + on the top would work fine.